src/app/admin/usersTable/UsersTable.tsx

```diff
@@ -224,10 +226,20 @@ const UsersTable: React.FC = () => {
                     ${error.logId}`);
                         break;
                 }
-            } else {
-                setUsers(data.userData);
-                setFilteredUsersCount(data.count);
+                return;
```

Comment:
Does this mean if there's any errors in the fetch, we return from fetchAndDisplayUserData without setting isLoading to false etc. (line 244 & 245)?

Reply:
You're right. That's not what we want to do. Have fixed